MISC/SetGravityLevel.md

```diff
@@ -9,11 +9,15 @@ void SET_GRAVITY_LEVEL(int level);
 ```
 
 ```
-level can be from 0 to 3  
```

Comment:
Wouldn't it be better to simply improve the native description and overall documentation for this native, so it's marked as 'Has Modern Docs'? As in actually write a description outside code blocks, you may want to check the [contribution guide](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md) by the way 😃.